features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/root/CreateRoomRootView.kt

```diff
@@ -77,7 +81,11 @@ fun CreateRoomRootView(
         ) {
             UserListView(
                 modifier = Modifier.fillMaxWidth(),
-                state = state.userListState,
+                // Do not render suggestions in this case, the suggestion will be rendered
+                // by CreateRoomActionButtonsList
+                state = state.userListState.copy(
+                    recentDirectRooms = persistentListOf(),
+                ),
```

Comment:
I do not really like to touch the state in the View, but adding another boolean like `renderSuggestions` to `UserListView` was more intrusive... Let me know what you reviewer(s) think, there are maybe other more elegant option.

Reply:
I also though about the option to enhance `UserListPresenterArgs`, but it does not work, the presenter is the same for both cases: suggestion for rooms and suggestion for users.

I will keep the current code, we may revisit later.

Reply:
Is it the same presenter? I see one being instantiated in `CreateRoomRootPresenter` and another one in `AddPeoplePresenter`.

Reply:
Yes, you're right.
Here the suggestions have to be in the state (so the presenter has to provide them, that's why I wanted to say by "the presenter is the same"), but are rendered in `CreateRoomActionButtonsList` and not in `UserListView`, from the same method (`CreateRoomRootView`).